test/uniswapTrades.test.ts

```diff
@@ -251,6 +251,23 @@ describe('Uniswap', () => {
       expect(methodParameters.value).to.eq(methodParametersV2.value)
     })
 
+    it('encodes a single exactOutput ETH->USDC swap, with a flat fee', async () => {
```

Comment:
maybe can add an exactOut, eth out test since we have that edge case where the fee is taken in WETH just to make sure its working with unwrap logic